examples/06-verif-manual/vm-008-parametric_calculation.py

```diff
@@ -0,0 +1,216 @@
+                    font_size=26)
+        return dist_kp, mapdl.klist()
+
+    def create_node_method(self):
+
+        # Define nodes by coordinates.
+        node1 = mapdl.n(node=1, x=self.x1, y=self.y1, z=self.z1)
+        node2 = mapdl.n(node=2, x=self.x2, y=self.y2, z=self.z2)
+
+        # Get the distance between nodes.
+        dist_node, node_x, node_y, node_z = mapdl.ndist(node1, node2)
+
+        # Plot nodes.
+        mapdl.nplot(nnum=True,
+                    vtk=True,
+                    color="grey",
+                    show_bounds=True,
+                    font_size=26)
+        return dist_node, mapdl.nlist()
+
+
+###############################################################################
+# Distance between keypoints
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Using already created method for keypoints to get the distance between them
+# and print out an output. The keypoints have got next coordinates:
+#
+# * :math:`K_{\mathrm{3(x,y,z)}} = 100, 0, 30`
+# * :math:`K_{\mathrm{4(x,y,z)}} = -200,25,80`
+
+kp = Create(100, 0, 30, -200, 25, 80)
+kp_dist, keypoint_list = kp.create_kp_method()
+print(f"Distance between keypoint is: {kp_dist:.2f}\n\n"
+      f"{keypoint_list}")
+
+
+###############################################################################
+# Distance between nodes.
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Using already created method for nodes to get the distance between them and
+# print out an output. The nodes have got next coordinates:
+#
+# * :math:`N_{\mathrm{1(x,y,z)}} = 1.5, 2.5, 3.5`
+# * :math:`N_{\mathrm{2(x,y,z)}} = -3.7, 4.6, -3`
+
+nodes = Create(1.5, 2.5, 3.5, -3.7, 4.6, -3)
+node_dist, node_list = nodes.create_node_method()
+print(f"Distance between nodes is: {node_dist:.2f}\n\n"
+      f"{node_list}")
+
+
+###############################################################################
+# Check Results
+# ~~~~~~~~~~~~~
+# Finally we have the results of the distances for both simulations,
+# which can be compared with expected target values:
+#
+# - 1st simulation to get the distance between keypoints :math:`K_3` and :math:`K_4`, where :math:`LEN_1 = 305.16\,(in)`
+# - 2nd simulation to get the distance between nodes :math:`N_1` and :math:`N_2`, where :math:`LEN_2 = 8.58\,(in)`
+#
+# For better representation of the results we can use ``pandas`` dataframe
+# with following settings below:
+
+# Define the names of the rows.
+row_names = ["N1 - N2 distance (LEN2)",
+             "K3 - K4 distance (LEN1)"]
+
+# Define the names of the columns.
+col_names = ['Target',
+             'Mechanical APDL',
+             'RATIO']
+
+# Define the values of the target results.
+target_res = np.asarray([8.5849, 305.16])
+
+# Create an array with outputs of the simulations.
+simulation_res = np.asarray([node_dist, kp_dist])
+
+# Identifying and filling corresponding columns.
+main_columns = {
+    "Target": target_res,
+    "Mechanical APDL": simulation_res,
+    "Ratio": list(np.divide(simulation_res, target_res))
```

Comment:
```suggestion
    "Ratio": list(simulation_res/target_res)
```

Since both are numpy arrays, I do not think you need to call ``np.divide``. I'm not even sure you need to convert it to a list. But maybe.

Reply:
just left the same style to show that numpy works)